src/oic/utils/sdb.py

```diff
@@ -1018,44 +884,49 @@ def get_client_id_for_session(self, sid):
 
     def get_client_ids_for_uid(self, uid: str) -> List[str]:
         """Return client_ids for a given uid."""
-        return self._db.get_client_ids_for_uid(uid)
+        return [self.get_client_id_for_session(sid) for sid in self._db.get_by_uid(uid)]
 
-    def get_verify_logout(self, uid: str) -> str:
-        """Return the key for logout verification."""
-        return self._db.get_verified_logout(uid)
+    def get_verify_logout(self, uid: str) -> Dict:
+        """Return the dictionary for logout verification."""
+        res = {}
+        for sid in self._db.get_by_uid(uid):
+            _dict = self._db[sid]
+            try:
```

Comment:
This can be rewritten using `get()`.

The except part is not tested.

Reply:
This is a provider issue. Not part of my PR.
So let's get back to it when we do the next part.

Reply:
If it is not part of PR, then it shouldn't be here. Unfortunately these things rarely get revisited... :)

Reply:
Docstring doesn't match the original implementation.

Reply:
Well, it matches the underlying method and is correct.

Reply:
I am talking about `get_verify_logout`, which has a return of `str` and the docstring claims that it returns `dictionary`.

Reply:
OK